docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md

```diff
@@ -74,3 +74,9 @@ Open Enclave SDK binary packages can also be [downloaded from GitHub](https://gi
 ### 4. Verify the Open Enclave SDK install
 
 See [Using the Open Enclave SDK](Linux_using_oe_sdk.md) for verifying and using the installed SDK.
+
+### 5. Determine call path for SGX quote generation in attestation sample
+
+In the attestation sample, you can either take the in-process call path or out-of-process call path to generate evidence of format `OE_FORMAT_UUID_SGX_ECDSA`. Please refer to the following README file for more information:
+
+ - [The Attestation Sample](samples/attestation/README.md)
```

Comment:
Can you link to the heading under the sample using README.md#inproc etc?

Reply:
Sure. I also changed `samples/...` to `/samples/...`.